### ISSUE TITLE
LOG wird mit INFO Meldungen geflutet

### ISSUE THREAD

**Issue by andre-bu** [truegreyhound/ioBroker.husq-automower#32]
Das LOG wird im Sekundentakt mit dieser Infomeldung geflutet:
"updateStatus mower object is NULL"

**Comment by reutli** [comment]
Jupp Adapter aktuell "tot", weil Husqvarna die alte API geschlossen hat. Siehe auch [https://github.com/truegreyhound/ioBroker.husq-automower/issues/31](url)

**Comment by andre-bu** [comment]
Stimmt, der ganze Adapter ist tot. Hab meinen Moe heute in wieder in Betrieb genommen. Na so ein Mist.

**Comment by gemo-bi** [comment]
Schade. Hat im Vorjahr wirklich gut funktioniert. Stopp on rain. :(
Gibt's hier Hoffnung auf etwas neues?

**Comment by andre-bu** [comment]
Es gibt inzwischen einen neuen Adapter, der mit der neuen API Version
funktioniert.

https://github.com/ice987987/ioBroker.husqvarna-automower



gemo-bi ***@***.***> schrieb am Mo., 18. Apr. 2022, 16:41:

**Comment by MatthiasUlrich84** [comment]
Auch hier wird sich eventuell noch was tun. Der Entwickler wollte eventuell über Ostern was daran machen.
